sub_cal/sub_cal/src/test/java/model/HistoryEntityTest.java

```diff
@@ -0,0 +1,41 @@
+package model;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class HistoryEntityTest {
+    private static HistoryEntity historyEntity = new HistoryEntity();
+
+    @BeforeEach
+    void setHistoryEntity(){
+        historyEntity = new HistoryEntity();
+        historyEntity.addHistory("1 + 1","2");
+        historyEntity.addHistory("1 * 3 + 6","9");
+    }
+
+    @Test
+    void getHistory() {
+        historyEntity.getHistory();
+        List<String> historyList = new ArrayList<>();
+        historyList.add("1 + 1 = 2");
+        historyList.add("1 * 3 + 6 = 9");
+        assertEquals(historyList,historyEntity.getHistory());
```

Comment:
[nit]
```suggestion
				// given
        List<String> expectedList = new ArrayList<>();
        expectedList.add("1 + 1 = 2");
        expectedList.add("1 * 3 + 6 = 9");
        
        // when
        List<Sring> result = historyEntity.getHistory();
        
        // then
        assertEquals(expectedList, result);
```

위와 같은 형태로 작성해보는 건 어떨까요?

Reply:
해당 테스트 코드 수정하였습니다 !
앞으로도 테스트코드를 짤 때 given when then 으로 작성해보도록 노력하겠습니다

리뷰 남겨주셔서 감사합니다
말씀해주신 부분들 참고해서 리팩토링 진행하겠습니다